Look for *.mo files in /usr/share/locale (FHS compliant)
--- a/mercurial/i18n.py
+++ b/mercurial/i18n.py
@@ -15,7 +15,7 @@
     module = __file__
 
 base = os.path.dirname(module)
-for dir in ('.', '..'):
+for dir in ('.', '..', '/usr/share'):
     localedir = os.path.join(base, dir, 'locale')
     if os.path.isdir(localedir):
         break
--- a/setup.py
+++ b/setup.py
@@ -280,7 +280,7 @@
         extmodules.append(inotify)
         packages.extend(['hgext.inotify', 'hgext.inotify.linux'])
 
-packagedata = {'mercurial': ['locale/*/LC_MESSAGES/hg.mo']}
+packagedata = {'mercurial': []}
 
 def ordinarypath(p):
     return p and p[0] != '.' and p[-1] != '~'
